nasa/harmony
# Harmony 1278 output file size

## Jira Issue ID
HARMONY-1278

## Description
Adds  dB migrations and code to lookup/store granule sizes to support batching.

## Local Test Steps
* Deploy to the sandbox
* Issue a harmony query to deployed environment
* Connect to dB and select work items
* Verify that the work items have a value for the outputGranuleSizes column

## PR Acceptance Checklist
